packages/neuron-wallet/src/services/cells.ts

```diff
@@ -526,6 +526,87 @@ export default class CellsService {
       anyoneCanPayOutputs,
       changeOutput,
       finalFee: finalFee.toString(),
+      sendCapacity: capacityInt.toString(),
+    }
+  }
+
+  public static async gatherAnyoneCanPaySendAllCKBInputs(
+    anyoneCanPayLockHashes: string[],
+    fee: string = '0',
+    feeRate: string = '0',
+    baseSize: number = 0,
+  ) {
+    const feeInt = BigInt(fee)
+    const feeRateInt = BigInt(feeRate)
+    const mode = new FeeMode(feeRateInt)
+
+    let needFee = BigInt(0)
+
+    // only live cells, skip which has data or type
+    const anyoneCanPayLockHashBuffers: Buffer[] = anyoneCanPayLockHashes.map(h => Buffer.from(h.slice(2), 'hex'))
+    const anyoneCanPayLockLiveCellEntities: LiveCellEntity[] = await getConnection()
+      .getRepository(LiveCellEntity)
+      .find({
+        where: {
+          lockHash: In(anyoneCanPayLockHashBuffers),
+          typeHash: null,
+          usedBlockNumber: null,
+        },
+      })
+    const anyoneCanPayLockLiveCells: LiveCell[] = anyoneCanPayLockLiveCellEntities.map(entity => LiveCell.fromEntity(entity))
+
+    if (anyoneCanPayLockLiveCellEntities.length === 0) {
+      throw new CapacityNotEnough()
+    }
+
+    const inputs: Input[] = []
+    const inputOriginCells: LiveCell[] = []
+    let inputCapacities: bigint = BigInt(0)
+    let totalSize: number = baseSize
+    anyoneCanPayLockLiveCells.every(cell => {
```

Comment:
The returned value is fixed to `true` and not used so `Array#forEach` is more proper than `Array.every`

Reply:
ok

Reply:
updated